applications/machine_learning/configuration/thingy53_nrf5340_cpuapp/dts.overlay

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright (c) 2021 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
```

Comment:
btw. @nvlsianpu, should we use Apache license here?
The file was actually copy-pasted from `nrf/modules/mcuboot/flash_sim.overlay`, so I used the same license identifier.

Reply:
Already discussed with @de-nordic and @carlescufi. The license was updated to SPDX-License-Identifier: LicenseRef-Nordic-5-Clause